src/modules/DeleteCompleted.js

```diff
@@ -0,0 +1,6 @@
+const deleteCompleted = (newTodoList) => {
+  newTodoList.removeTodoItemByCompleted();
+  // generateListHTML(newTodoList);
+};
+
```

Comment:
- [ ] I think this would work better directly in the file you call it instead of creating a new file.